### ISSUE TITLE
Please support emitting CollectD metrics

### ISSUE THREAD

**Issue by seitzbg** [PowerDNS/pdns#2590]
Please support emitting CollectD metrics emitting.

carbon: 
lieter | phibs: [graphite] 2015/06/12 17:12:19 unable to parse data: received "pdns.influx-grafana-test.auth.udp6-queries" which doesn't conform to format of key.value.key.value.name or name 

lieter | I did some work on the collectd plugin for logstash some time ago
lieter | phibs: maybe we should hack in collectd metrics :D
lieter | phibs: can you make a ticket?

**Comment by rubenk** [comment]
collectd has quite a nice pdns plugin for what it's worth ;)

**Comment by pieterlexis** [comment]
yes, this is true. But emitting for e.g. [influxdb](http://influxdb.com) and [prometheus](http://prometheus.io/) (both HTTP) would be cool as well.

**Comment by seitzbg** [comment]
Also not needing collectd if I don't already use it for other metrics, priceless ;)

**Comment by janeczku** [comment]
@pieterlexis In the meantime: https://github.com/janeczku/powerdns_exporter
